pom.xml

```diff
@@ -14,7 +14,7 @@
     <packaging>pom</packaging>
 
     <name>Test suite for MicroProfile on WF/EAP</name>
-    
+
     <modules>
         <module>tooling-cpu-load</module>
         <module>tooling-docker</module>
```

Comment:
What about `tooling-docker`? I don't see any updates for this module. Are tests from this module executed in a same way like without bootable profile?

Reply:
I am not against excluding something but this module is not tested like the other ones (involving Arquillian) but just two Wildfly containers.
Indeed, this is the only module between the `tooling-*` ones which didn't require for the testing to be adapted in order to prevent problems with Arquillian. 
Basically the `tooling-*` modules are tested only with the standard TS execution.

Reply:
tooling-docker doesn't contain any arq tests AFAIK, resolving, thank you.